modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -1,10 +1,10 @@
 = Domain Support for Mule 4 API Proxies
 
-Mule runtime engine enables you to share common resources among other Mule applications by defining these resources within a Mule domain. Other Mule applications can then inherit this domain.
+Mule runtime engine (Mule) enables you to share common resources among other Mule applications by defining these resources within a Mule domain. Other Mule applications can then inherit this domain.
 
 This way, your Mule applications that are configured for it, can access the xref:4.1@mule-runtime::shared-resources.adoc[resources]declared in your domain. +
```

Comment:
"it" has no clear antecedent. I'm not sure what "This way" or "it" refers to.

Reply:
Done! Thanks!